src/Graph.RBAC/Graph.RBAC/Microsoft.Azure.Graph.RBAC.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.Azure.Graph.RBAC
     -->
     <SdkNuGetPackage Include="Microsoft.Azure.Graph.RBAC">
-      <PackageVersion>1.8.0-preview</PackageVersion>
+      <PackageVersion>2.0.0-preview</PackageVersion>
```

Comment:
version 2.0.0 of Graph library already exists. it is AutoRest based one. You should keep Hyak based version < 2.0. Please make it 1.9.0-preview